src/appkit.cs

```diff
@@ -12674,7 +12679,7 @@ public interface NSSlider : NSAccessibilitySlider {
 		IntPtr Constructor (CGRect frameRect);
 
 		[Export ("isVertical")]
-		nint IsVertical { get; [Mac (10, 12)] set; }
+		nint IsVertical { get; /* [Mac (10, 12)] set; -- Radar 27222357 */ }
```

Comment:
Of course - https://trello.com/c/EbOXcErP/48-27222357-nsslider-isvertical-changed-sized

I'll add the FIXME